static/src/components/builder/components/TextTagging/TextTaggingForm.vue

```diff
@@ -131,7 +135,7 @@
       class="btn btn-default btn-sm col-sm-2"
       @click="addTag"
     >
-      Add Tag
+      New Tag
```

Comment:
It looks good to me, and you don't have to change this but I prefer it to be `Add More Tags`.

Reply:
How about `Add New Tag`, because it's only adding 1 tag. `Add More Tags` sounds like it will add more than 1.